File: countZero.cs
public int CountZeros(int[] A)
{
    var arraylistValues = new List<int>();
    for(int i=0; i< A.Length, i++) { |

        if (!arraylistValues.Contains(A[i]) and (A[i].Equals(0))
        {
            arraylistValues.Add(A[i])
        }
            
    }
    return arraylistValues.Count

}

public int getX(double a, double b, double c)
{
    
    double x1, x2 = 0.0;
    x1 = (((-1) * b) + (Math.Sqrt(Math.Sqr(b) - (4 * a * c))))/(2*a);
    x2 = (((-1) * b) - (Math.Sqrt(Math.Sqr(b) - (4 * a * c)))) / (2 * a);

    if (x1 > x2)
    {
        return x1;
    }
    else
    {
        return x2
    }
}

File: count_zero.py
from flask import Flask
import sys, json

app = Flask(__name__)

'''a function that counts numbers of zeros in a given array'''

def countZeros():
    '''Array A contains driver values'''
    A = ['0', '0', '0', '1', '2', '2']
    counts = A.count('0')
    return ({"number of zeros":counts})


'''same method not inside a function, counts numbers of zeros in a given array'''
A = ['0', '0', '9', '1', '2', '2']
count = A.count('0')
print(count)

if __name__ == "__main__":
    app.run(debug=True)
    # A = ['0','0','0','1','2','2']
